packages/templating/plugin/html_scanner.js

```diff
@@ -169,8 +171,15 @@ html_scanner = {
           templateDotNameLiteral + ", " + renderFuncCode + ");\n";
       } else {
         // <body>
-        if (hasAttribs)
-          throwParseError("Attributes on <body> not supported");
+        if (hasAttribs) {
+          for(var attr in attribs) {
```

Comment:
Nope so the plugin doesn't get underscore unless explicitly asked for the in the RegisterBuildPlugin. So I think it's not really needed only for this loop...